docs/how-to-guides/develop-a-flow/develop-chat-flow.md

```diff
@@ -0,0 +1,61 @@
+# Develop Chat flow
+
+:::{admonition} Experimental feature
+This is an experimental feature, and may change at any time. Learn [more](../faq.md#stable-vs-experimental).
+:::
+
+From this document, you can learn how to develop a chat flow by writing a flow yaml from scratch. You can 
+find additional information about flow yaml schema in [Flow YAML Schema](../../reference/flow-yaml-schema-reference.md).
+
+## Flow input data
+
+The most important elements that differentiate a chat flow from a standard flow are **Chat Input** and **Chat History**. A chat flow can have multiple inputs, but **Chat History** and **Chat Input** are required inputs in chat flow.
+
+- **Chat Input**: Chat input refers to the messages or queries submitted by users to the chatbot. Effectively handling chat input is crucial for a successful conversation, as it involves understanding user intentions, extracting relevant information, and triggering appropriate responses.
+
+- **Chat History**: Chat history is the record of all interactions between the user and the chatbot, including both user inputs and AI-generated outputs. Maintaining chat history is essential for keeping track of the conversation context and ensuring the AI can generate contextually relevant responses. Chat History is a special type of chat flow input, that stores chat messages in a structured format.
```

Comment:
we should have some examples showing the format of chat history

Reply:
Add example to show the format of chat history